include/config.h

```diff
@@ -83,7 +83,14 @@
    will be kept and written to the crash/ directory as RECORD:... files.
    Note that every crash will be written, not only unique ones! */
 
-// #define AFL_PERSISTENT_RECORD
+#define AFL_PERSISTENT_RECORD
```

Comment:
this has a fuzzing overhead, so I would not want to enable it by default

Reply:
Ack! This slipped in inadvertently because of the final testing round, gonna comment it.